.github/workflows/publish.yml

```diff
@@ -0,0 +1,29 @@
+name: Publish
+
+on:
+  release:
+    types:
+      - created
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Install Poetry
+        run: |
+          python -m pip install --upgrade poetry wheel
+      - name: Install dependencies
+        run: |
+          poetry install
+      - name: Publish
+        env:
+          POETRY_PYPI_TOKEN_PYPI: ${{ secrets.POETRY_PYPI_TOKEN_PYPI }}
+        run: |
+          poetry config pypi-token.pypi $POETRY_PYPI_TOKEN_PYPI
+          poetry build
+          poetry publish --dry-run
```

Comment:
Will the `--dry-run` be removed after validating that this workflow triggers and acts as expected?

Reply:
good catch, will be removed